jenkins/PerfCI_Chaos/05_PerfCI_Chaos_Verify_VMs_Deployment/Jenkinsfile

```diff
@@ -0,0 +1,225 @@
+                    }
+                }
+            } // Fedora Verifications
+
+            stage('Fedora Cleanup') {
+                steps {
+                    script {
+                            sh """
+                                sudo podman run --rm -t \
+                                -e WORKLOAD='bootstorm_vm' \
+                                -e KUBEADMIN_PASSWORD='${KUBEADMIN_PASSWORD}' \
+                                -e ELASTICSEARCH='${ELASTICSEARCH}' \
+                                -e ELASTICSEARCH_PORT='${ELASTICSEARCH_PORT}' \
+                                -e IBM_REGION_NAME='${IBM_REGION_NAME}' \
+                                -e IBM_ENDPOINT_URL='${IBM_ENDPOINT_URL}' \
+                                -e IBM_ACCESS_KEY_ID='${IBM_ACCESS_KEY_ID}' \
+                                -e IBM_SECRET_ACCESS_KEY='${IBM_SECRET_ACCESS_KEY}' \
+                                -e IBM_BUCKET='${IBM_BUCKET}' \
+                                -e IBM_KEY='${IBM_KEY}' \
+                                -e GOOGLE_DRIVE_PATH='${GOOGLE_DRIVE_PATH}' \
+                                -e GOOGLE_DRIVE_CREDENTIALS_FILE='${GOOGLE_DESTINATION_PATH}/${GOOGLE_CREDENTIALS_FILE}' \
+                                -e GOOGLE_DRIVE_TOKEN_FILE='${GOOGLE_DESTINATION_PATH}/${GOOGLE_TOKEN_FILE}' \
+                                -e GOOGLE_DRIVE_SHARED_DRIVE_ID='${GOOGLE_DRIVE_SHARED_DRIVE_ID}' \
+                                -e RUN_ARTIFACTS_URL='${RUN_ARTIFACTS_URL}' \
+                                -e RUN_TYPE='${RUN_TYPE}' \
+                                -e TIMEOUT='${TIMEOUT}' \
+                                -e log_level='INFO' \
+                                -v '${KUBECONFIG_PATH}:${KUBECONFIG_PATH}' \
+                                -v '${TEMP_DIR}:${GOOGLE_DESTINATION_PATH}' \
+                                --privileged '${QUAY_BENCHMARK_RUNNER_REPOSITORY}'
+                                """
+                    }
+                }
+            } // Fedora Cleanup
+
+    } // stages
+
+    post {
+        always {
+            script {
+                sh '''
+                    sudo podman ps -a --filter "name=benchmark-runner" -q | xargs -r podman rm -f
+                    # Clean up the virtual environment
+                    rm -rf venv
+                    # Clean temp dir
+                    if [ -d "${TEMP_DIR}" ]; then
+                        sudo rm -rf "${TEMP_DIR}"
+                        echo "Temporary directory '${TEMP_DIR}' deleted."
+                    fi
+                '''
+            }
+        }
+        failure {
+            script {
+                sh """ sudo podman run --rm -e KUBEADMIN_PASSWORD='${KUBEADMIN_PASSWORD}' -e ELASTICSEARCH='${ELASTICSEARCH}' -e ELASTICSEARCH_PORT='${ELASTICSEARCH_PORT}' -e CI_STATUS=''failed'' -e RUN_TYPE='${RUN_TYPE}' -e log_level='INFO' -v '${KUBECONFIG_PATH}:${KUBECONFIG_PATH}' --privileged '${QUAY_BENCHMARK_RUNNER_REPOSITORY}' """
```

Comment:
This should also have the multiline syntax fixed.

Also, you have two single quotes in `CI_STATUS=''failed''` which needs to be fixed.